src/components/passphraseCreation/index.test.js

```diff
@@ -63,7 +63,7 @@ describe('Passphrase Creation', () => {
     });
   });
 
-  describe('Using mobile device', () => {
+  describe.skip('Using mobile device', () => {
```

Comment:
I assume it is skipped because we currently only have a desktop version. Can you put there a comment that says why it's skipped?